config/config.go

```diff
@@ -58,6 +58,7 @@ const (
 	COMPOSE_HTTP_TIMEOUT                 = "launchtask.composehttptimeout"
 	HTTP_TIMEOUT                         = "launchtask.httptimeout"
 	COMPOSE_STOP_TIMEOUT                 = "cleanpod.timeout"
+	DEFAULT_COMPOSE_STOP_TIMEOUT         = 20
```

Comment:
why is this changed from 10?

Reply:
The dce readme and config file says default 20. Hence changed. However, the default docker stop timeout is 10. Should I revert?

Reply:
i will let @JiaminZhu decide then

Reply:
timeout can be changed through config file, but let's keep the default as 10

Reply:
i still see it as 20 and it should be `10s`